### PR TITLE
add zhiCN.playground for chinese

### PR DESCRIPTION
**Pull request by wangrui460**
This repository is so great

### CODE REVIEW

**Comment by wangrui460** [comment]
i don't know how to change

File: Sample-iOS-zhiCN.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+## Injection of Value s     通过 值 注入
+*/
+
+struct Turtle: Animal {
+    var name: String?
+    
+    init(name: String?) {
+        self.name = name
+    }
+
```
[the remark this answers is not in the document]
Comment (wangrui460) [reply Sample-iOS-zhiCN.playground/Contents.swift]: The same as the original

**Comment by jakubvano** [comment]
Thanks for the PR! 👍 

You can fix most of the swiftlint issues by running `swiftlint autocorrect` (if you are not familiar with it, check out https://github.com/realm/SwiftLint).

Trailing whitespace issue can be prevented by Xcode settings (Preferences -> Text Editing -> Automatically trim trailing whitespace)

**Comment by jakubvano** [comment]
Closing due to inactivity

**Pull request closed without merging**